Yosemite/Yosemite/Tools/Media/MediaAssetExporter.swift

```diff
@@ -50,7 +50,8 @@ final class MediaAssetExporter: MediaExporter {
         if let resource = resources.first {
             resourceAvailableLocally = true
             filename = resource.originalFilename
-            if UTTypeEqual(resource.uniformTypeIdentifier as CFString, kUTTypeGIF) {
+            let gifTypeIdentifier = UTType.gif.identifier as CFString
+            if UTTypeEqual(resource.uniformTypeIdentifier as CFString, gifTypeIdentifier) {
```

Comment:
nit: I think comparing the identifier strings directly should be fine?

```suggestion
            if resource.uniformTypeIdentifier == UTType.gif.identifier {
```